Lib/test/test_types.py

```diff
@@ -756,6 +757,23 @@ def __module__(self):
         with self.assertRaises(ZeroDivisionError):
             str | TypeVar()
 
+    @cpython_only
+    def test_or_type_operator_reference_cycle(self):
+        if not hasattr(sys, 'gettotalrefcount'):
+            self.skipTest('Cannot get total reference count.')
+        gc.collect()
+        before = sys.gettotalrefcount()
+        for _ in range(30):
+            T = typing.TypeVar('T')
+            U = int | list[T]
+            T.blah = U
+            del T
+            del U
+        gc.collect()
+        leeway = 64
+        self.assertLessEqual(sys.gettotalrefcount(), before + leeway,
```

Comment:
I think that the error report will be more readable if compare delta:
```suggestion
        self.assertLessEqual(sys.gettotalrefcount() - before, leeway,
```
At least it is written in such way in other tests.